packages/wp-now/src/download.ts

```diff
@@ -2,21 +2,26 @@ import fs from 'fs-extra';
 import path from 'path';
 import https from 'https';
 import unzipper from 'unzipper';
+import os from 'os';
 import { IncomingMessage } from 'http';
 import {
+	DEFAULT_WORDPRESS_VERSION,
 	SQLITE_PATH,
 	SQLITE_URL,
 	WORDPRESS_VERSIONS_PATH,
-	WP_DOWNLOAD_URL,
 	WP_NOW_PATH,
 } from './constants';
 
+function getWordPressVersionUrl(version = DEFAULT_WORDPRESS_VERSION) {
+  return `https://wordpress.org/wordpress-${version}.zip`;
```

Comment:
the permissiveness here might be useful, but could also lead to some needless errors from the URL being wrong, e.g. `--wp v6.3-rc2`

if we drew a line and claimed that we only support the numeric version numbers with decimal separators, we could validate the version before attempting to fetch it.

```js
if ( ! /^latest|(?:[1-9]\d*\.\d+)$/.test( version ) ) {
	throw new ValueError( 'Only recognizes "latest" and versions like "6.2"' );
}
```

Reply:
Thanks for this suggestion. I included that validation here: https://github.com/WordPress/wordpress-playground/pull/252/commits/c44033ee269da404d28a3556ff20492f34f688b9

I also accept beta versions such as `6.2-beta1` and `6.2-RC1`.
I created the Regex based on the releases on https://wordpress.org/download/releases/#betas
I also added tests for this function, and the first texts in `wp-now`